src/xmipp/applications/programs/mpi_nma_alignment_vol/mpi_nma_alignment_vol_main.cpp

```diff
@@ -0,0 +1,140 @@
+/***************************************************************************
+ *
+ * Authors:  Slavica Jonic slavica.jonic@impmc.jussieu.fr
```

Comment:
Who is the author? Slavica Jonic or you?

Reply:
This was written by both of us, I forgot to add my name.